paddle/fluid/framework/ir/graph_helper.h

```diff
@@ -24,6 +24,9 @@ limitations under the License. */
 namespace paddle {
 namespace framework {
 namespace ir {
+// OpDesc In sequence
+constexpr char kAllOpDescs[] = "all_op_descs";
```

Comment:
Dont put this in graph_helper.h. move them to your pass.cc

Reply:
ok.